lib/views/help/report_a_data_breach.html.erb

```diff
@@ -0,0 +1,92 @@
+<h1>Report a data breach</h1>
+
+<% @title = "Report a data breach" %>
+
+<%= render partial: 'sidebar' %>
+
+<div id="left_column_flip" class="left_column_flip">
+  <h2>Use this form to report a data breach</h2>
+
+  <%# TODO: High level description of what this page is about, and the key things to know about how we process these reports %>
+
+  <%= form_with model: @report, url: help_report_a_data_breach_handle_form_submission_path, html: { class: 'contact-form' } do |f| %>
+
+    <%= foi_error_messages_for :report %>
+
+    <p>
+      <%= f.label :url, "Please provide a link to the request or attachment containing the data breach." %>
+      <span class="form_item_note">
+        Alternatively you can provide the request's email address, if known.
+        <br>
+        <br>
+      </span>
+      <%= f.text_field :url %>
+    </p>
+
+    <p>
+      <label class="form_label">
+        What type of data breach is this?
+        <%= f.label :type_ico, class: 'form_inline' do %>
+          <%= f.radio_button :type, 'public_body_required_to_report_to_ico' %>
+          A data breach which a public body is required to report to the ICO
+        <% end %>
+
+        <%= f.label :type_data, class: 'form_inline' do %>
+          <%= f.radio_button :type, 'special_category_or_criminal_offence_data' %>
+          A data breach involving special category or criminal offence data
+        <% end %>
+
+        <%= f.label :type_other, class: 'form_inline' do %>
+          <%= f.radio_button :type, 'other' %>
+          Other
+        <% end %>
+      </label>
+    </p>
+
+    <p>
+      <%= f.label :message, "Please provide details about this data breach" %>
+      <%= f.text_area :message, rows: 3 %>
+    </p>
+
+    <p>
+      <%= f.label :contact_email, "Please provide the email address for the Data Protection Officer (if known):" %>
+      <%= f.text_field :contact_email %>
+    </p>
+
+    <p>
+      <label class="form_label">
+        Are you reporting on behalf of the public body responsible for the data breach?
+        <%= f.label :is_public_body_yes, class: 'form_inline' do %>
+          <%= f.radio_button :is_public_body, true %>
+          Yes
+        <% end %>
+        <%= f.label :is_public_body, class: 'form_inline' do %>
+          <%= f.radio_button :is_public_body, false %>
+          No
+        <% end %>
+      </label>
+    </p>
+
+    <div class="actions">
+      <%= f.submit "Send" %>
+    </div>
+
+  <% end %>
```

Comment:
```suggestion
    </div>
    <%= render :partial => "contact_form_privacy_notice" %>

  <% end %>
```

Suggest adding the "what we'll do with your data" partial (depends on #1720).

Reply:
Added